src/c++/perf_analyzer/genai-perf/genai_perf/profile_data_parser/profile_data_parser.py

```diff
@@ -26,13 +26,22 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from enum import Enum, auto
 from pathlib import Path
 from typing import List, Tuple
 
-from genai_perf.metrics import Metrics, ResponseFormat, Statistics
+from genai_perf.metrics import Metrics, Statistics
 from genai_perf.utils import load_json
 
 
+class ResponseFormat(Enum):
+    OPENAI_CHAT_COMPLETIONS = auto()
+    OPENAI_COMPLETIONS = auto()
+    OPENAI_EMBEDDINGS = auto()
+    OPENAI_RANKINGS = auto()
```

Comment:
Would you be able to update this to just be RANKINGS? OpenAI has no API for rankings.